cmd/gateway/Dockerfile

```diff
@@ -5,12 +5,12 @@ FROM storjlabs/golang as build-env
 
 # final stage
 FROM alpine
-ENV CONF_PATH=/root/.local/share/storj/uplink \
+ENV CONF_PATH=/root/.local/share/storj/gateway \
```

Comment:
I suspect that these may have been intentionally using the same location to ensure that they use the same identity.